src/mpj-App.js

```diff
@@ -183,9 +183,7 @@ class App extends Component {
             <Playlist playlist={playlist} index={i} />
           )}
         </div> : <button onClick={() => {
-            window.location = window.location.href.includes('localhost') 
-              ? 'http://localhost:8888/login' 
-              : 'https://better-playlists-backend.herokuapp.com/login' }
+            window.location = 'https://better-playlists-backend.herokuapp.com/login' }
```

Comment:
Oh yeah btw what is this file? Does this ever get executed or were you basing everything else off this and its not used?

Reply:
The latter. It's from a YouTube tutorial that I based much of the initial code on for the authentication flow and requisite API calls.